Src/Newtonsoft.Json/Linq/JContainer.cs

```diff
@@ -273,7 +273,7 @@ public override JEnumerable<JToken> Children()
         /// <returns>
         /// A <see cref="IEnumerable{T}"/> containing the child values of this <see cref="JToken"/>, in document order.
         /// </returns>
-        public override IEnumerable<T> Values<T>()
+        public override IEnumerable<T?> Values<T>() where T : default
```

Comment:
I'm not familiar with this constraint. Was this added in C# 9?

Reply:
Yes https://github.com/dotnet/csharplang/blob/9807f9b2e2105255c0a658f3afc908769bb31339/proposals/csharp-9.0/unconstrained-type-parameter-annotations.md#default-constraint

Reply:
@JamesNK Thanks for the reference, I appreciate it!